### PR TITLE
Fix sign up page email and screen name input size

### LINKED ISSUE #6011: Input text fields have different widths in Sign Up page
The `email-address` and `Choose a screen name` input elements are narrower in width than the other two input tags.

### Evidence / Screenshot (if possible)
Screenshot 2021-12-24 at 9 11 57 AM

### Relevant url?

https://openlibrary.org/account/create

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual:
* Expected:

### Details

- **Logged in (Y/N)?**Y
- **Browser type/version?**Chrome
- **Operating system?**OSX
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

The Issue can be solved either by reducing the widths of the other two input elements or increasing the widths of first two input elements.
### Stakeholders

@jimchamp

### PR DESCRIPTION
**Pull request by jollypolly123**
Closes #6011

hotfix

### Technical

Original implementation selected the incorrect input type (email) instead of text for email address and screen name fields on the signup page.

### Testing

Navigate to /account/create

### Screenshot

image

### CODE REVIEW

**Comment by mekarpeles** [comment]
Hi @jollypolly123! Thank you for submitting this fix!

@jimchamp I also wanted to note the solution here as it may be equally simple and be an approach we may wish to evaluate.
https://github.com/internetarchive/openlibrary/issues/6011#issuecomment-1013672237

Thank you both!

File: static/css/components/form.olform.less [hunk 1]
```diff
@@ -48,7 +48,7 @@
     width: 4em;
   }
 
-  input[type=email],
+  input[type=text],
   input[type=password],
   .required {
```
Comment (jimchamp) [inline static/css/components/form.olform.less:53]:
```suggestion
  .required {
```
Removing the input selectors solves this issue, and doesn't resize other fields in other views.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Happy to merge this when the requested changes have been made.

**Review by jimchamp — APPROVED** [review]
Lgtm!  Thanks, @jollypolly123!  Will merge when the checks pass.

**Pull request merged**